Source/MappingLogic.cpp

```diff
@@ -61,6 +61,7 @@ juce::Colour MappingLogicBase::indexToColour(int inx) const
 
 void MappingLogicBase::indexToTerpstraKey(int inx, TerpstraKey& keyData) const
 {
+    keyData.keyType = LumatoneKeyType::noteOnNoteOff;

```

Comment:
I was getting a bunch of disabled keys without this, but I'm not sure whether this is the correct place to set it (I didn't yet look into what else calls this method.)